Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml

```diff
@@ -225,6 +225,32 @@
   - type: Stack
     lingering: true
 
+- type: entity
+  parent: BaseHealingItem
+  id: Tourniquet
+  name: Tourniquet
```

Comment:
AHHHHH MY EYES, NAME FROM CAPITAL LETTER AHHHH

Reply:
time to fix